packages/bbui/src/Button/Button.svelte

```diff
@@ -32,7 +43,7 @@
   class:disabled
   class="spectrum-Button spectrum-Button--size{size.toUpperCase()}"
   {disabled}
-  on:click|preventDefault
```

Comment:
Preventing default on click here breaks form submission if you try and use this button as the submit input, this work around is an attempt to allow it to be used with forms